web/src/alert_words_ui.ts

```diff
@@ -39,16 +42,34 @@ export function rewire_rerender_alert_words_ui(value: typeof rerender_alert_word
     rerender_alert_words_ui = value;
 }
 
-function update_alert_word_status(status_text: string, is_error: boolean): void {
-    const $alert_word_status = $("#alert_word_status");
+const alert_word_status_banner = (alert_word: string, is_error: boolean): Banner => {
```

Comment:
Can this just be a function which calls `banners.open` instead of returing `Banner` object and then caller calling `banners.open`?

Reply:
Yep, that sounds better! Changed to a `open_alert_word_status_banner` function which opens the banner in the same function call.